extensions/converters/list/src/main/java/io/smallrye/config/converter/list/ArrayConverter.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.smallrye.config.converter.list;
+
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.eclipse.microprofile.config.spi.Converter;
+
+/**
+ * Converts a comma separated string to a String array
+ * @author <a href="mailto:phillip.kruger@redhat.com">Phillip Kruger</a>
+ */
+public class ArrayConverter implements Converter<String[]> {
```

Comment:
Arrays are already natively converted by SmallRye Config.  This native conversion happens in `io.smallrye.config.SmallRyeConfig#convert`.  It automatically uses the converter of the array element type to convert the values.

Reply:
Yes, this was not part of the spec initially, hence this converter. However now it's not needed anymore. I removed this.